fundamental-concepts/stdlib-tour/init_background.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+echo initializing...
+
+wget https://katacoda.com/drsk/courses/fundamental-concepts/data-structures-stdlib/assets/organizer.tar.gz
+
```

Comment:
```suggestion
wget https://katacoda.com/drsk/courses/fundamental-concepts/data-types-imports/assets/organizer.tar.gz

```

Reply:
sorry, forgot to push my changes to my master to this branch. this was fixed already, but thanks!